### ISSUE TITLE
Cannot find LV2 headers

### ISSUE THREAD

**Issue by falkTX** [rncbc/qtractor#185]
drobilla changed the way LV2 headers are installed in the system.
with latest lv2 project install from git, qtractor fails to find their headers.
```
checking lv2.h usability... no
checking lv2.h presence... no
checking for lv2.h... no
configure: WARNING: *** LV2 header files not found.
configure: WARNING: *** LV2 plug-in support will be disabled.
```

According to the build log, lv2-dev package is installed.
The lv2-dev package as packaged in kxstudio has the files as usual, see https://paste.debian.net/1057774/

I updated the entire LV2 stack a few days ago - that is, lv2, serd, sord, sratom, lilv and suil.
Afterwards I rebuild qtractor.
Somehow latest build of qtractor in kxstudio repos ships without LV2 support, even though nothing in the build process changed except updating the lv2 stack.

**Comment by rncbc** [comment]
qtractor configure expects lv2.h to be in /usr/include/lv2.h as was the usual location; moving it to /usr/include/lv2/lv2.h is new and not compliant with last but still current lv2 public release (v1.14.0).

as a possible workaround please try with `CFLAGS=-I/usr/include/lv2 ./configure` ...

cheers

**Comment by rncbc** [comment]
hopefully fixed on todays qtractor-0.9.4.8git.[1530e7](https://github.com/rncbc/qtractor/commit/1530e7f) .

**Comment by falkTX** [comment]
I can confirm it is fixed, builds with lv2 enabled again.
Thanks for the quick fix.
